### ISSUE TITLE
Load from nib not working

### ISSUE THREAD

**Issue by robyxz** [freak4pc/SMTagField#5]
initWithFrame is not called when control is loaded from a nib file. I ssuggest this update:

-(void)setup {
    self.backgroundColor        = [UIColor whiteColor];
    self.opaque                 = NO;
    self.autocorrectionType     = UITextAutocorrectionTypeNo;
    tags                        = @[];

```
tagsView                    = [[UIView alloc] initWithFrame: CGRectMake(0, 0, 0, 0)];
tagsView.backgroundColor    = [UIColor clearColor];

paddingView                 = [[UIView alloc] initWithFrame: CGRectMake(0, 0, 0, 20)];
paddingView.backgroundColor = [UIColor clearColor];
paddingView.opaque          = NO;

self.leftView               = paddingView;
self.leftViewMode           = UITextFieldViewModeAlways;

[[NSNotificationCenter defaultCenter] addObserver:self
                                         selector:@selector(textFieldDidChange:)
                                             name:@"UITextFieldTextDidChangeNotification"
                                           object:nil];

[[NSNotificationCenter defaultCenter] addObserver:self
                                         selector:@selector(textFieldDidEndEditing:)
                                             name:@"UITextFieldTextDidEndEditingNotification"
                                           object:nil];

[self addSubview: tagsView];
```

}
- (id)initWithFrame:(CGRect)frame
  {
  if (self = [super initWithFrame: CGRectMake(frame.origin.x, frame.origin.y, frame.size.width, 30)]) {
      [self setup];
  }
  
  return self;
  }

-(void)awakeFromNib {
    [self setup];
}

**Comment by freak4pc** [comment]
Hey @robyxz , I really appreciate it! I'm gonna add a similar patch soon.

**Comment by freak4pc** [comment]
Solved in commit de149d5 . Cheers !
